### PR TITLE
published operator v0.20.0

### PR DESCRIPTION
**Pull request by bart0sh**
This PR contains changes that were made to publish operator v0.20.0 on the operator hub: https://operatorhub.io/operator/intel-device-plugins-operator

It makes sense to also do these changes for release-0.20 branch in my opinion

### CODE REVIEW

**Review by mythi — DISMISSED** [review]

File: packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (rojkov) [inline packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: Do we really want to keep the generated packagemanifests in the repo?

I thought they were of no use in the repo and didn't include them for the 0.19 release.

Comment (bart0sh) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: They're quite useful for me to see the difference between previous and new versions, but this is minor. I can remove them if needed.

Comment (bart0sh) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: @mythi wdyt?

Comment (rojkov) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: The difference can be checked against the manifests published at the operatorhub.

Comment (bart0sh) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: True, but you have to copy manifests into the certain location of the community-operators clone to do that.

Comment (mythi) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml:1]: > I thought they were of no use in the repo and didn't include them for the 0.19 release.

@rojkov has a good point here. If it's just a copy of the CRD and clusterserviceversion that are in this repo already, I'd drop `packagemanifest/*`.

Comment (bart0sh) [reply packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml]: ok, dropped

**Review by rojkov — APPROVED** [review]

**Review by mythi — APPROVED** [review]

**Pull request merged**